core/templates/pages/topic-editor-page/editor-tab/topic-editor-tab.directive.ts

```diff
@@ -331,12 +331,18 @@ angular.module('oppia').directive('topicEditorTab', [
               newPracticeTabIsDisplayed) {
             if (
               newPracticeTabIsDisplayed !==
-              $scope.topic.getPracticeTabIsDisplayed()) {
+              $scope.topic.getPracticeTabIsDisplayed() && $scope.topicHasAtLeastTenPracticeQuestions()) {
               TopicUpdateService.setPracticeTabIsDisplayed(
                 $scope.topic, newPracticeTabIsDisplayed);
             }
           };
 
+          $scope.topicHasAtLeastTenPracticeQuestions = function() {
+            let skillQuestionCounts = Object.values($scope.skillQuestionCountDict);
+            let numberOfPracticeQuestions = skillQuestionCounts.reduce((a: number, b: number) => a + b, 0);
+            return numberOfPracticeQuestions >= 10;
```

Comment:
Replace the hardcoded number with a named constant.

Reply:
@Jeremy-Lemoine did push it!